.agents/skills/sym-fix-ci/scripts/inspect_pr_checks.py

```
@@ -0,0 +1,509 @@
+
+    if args.json:
+        print(json.dumps({"pr": pr_value, "results": results}, indent=2))
+    else:
+        render_results(pr_value, results)
+
+    return 1
+
+
+def find_git_root(start: Path) -> Path | None:
+    result = subprocess.run(
+        ["git", "rev-parse", "--show-toplevel"],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)